routes/users.js

```diff
@@ -41,7 +41,9 @@ router.post(
 
       user = new User({
         _id: req.user.uid,
-        postcode
+        postcode,
+        display_name: displayName,
+        profile_pic: photoURL
```

Comment:
We also need the email, right?

Reply:
Thought we could just pull that from the auth state as the user won't be able to update/ change it.